Kaggle/Titanic/titanic_analysis3.ipynb

```diff
@@ -34,11 +34,11 @@
    "execution_count": 1,
```

Comment:
<p>New chart added. how about that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ronald-leung/datascience_machinelearning/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>